server/controllers/apply/applicationsController.test.ts

```diff
@@ -698,4 +698,16 @@ describe('applicationsController', () => {
       })
     })
   })
+
+  describe('prisonDashboard', () => {
+    it('renders the prison dashboard page', async () => {
+      const requestHandler = applicationsController.prisonDashboard()
+
+      await requestHandler(request, response, next)
+
+      expect(response.render).toHaveBeenCalledWith('applications/prison-dashboard', {
+        pageHeading: 'Applications for prison',
```

Comment:
Tbh I don't know that we need this pageHeading at all, the template isn't being shared with other pages so I don't see why the Controller should be passing it in/there's nothing dynamic that needs to be calculated... will remove and if later tickets adding content to the page need it then it can be put back in